build.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# Expects the following variables to be defined.
+#
+# BUILDLOG_ROOT - directory for downloading and producing output
+# AUTOBUILD_ROOT - directory containing autobuild
+
+set -u
+
+SCOREBOARD_ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+echo "----- Starting mirror_scoreboard";
+
+if ! [ -d "${BUILDLOG_ROOT}" ]
+then
+    echo "BUILDLOG_ROOT (${BUILDLOG_DIR}) does not exist"
+    exit 1
+fi
+
+# Build the main selection page
+echo "----- Building Main Index page"
+"${AUTOBUILD_ROOT}/scoreboard.pl" -l -v -i "${SCOREBOARD_ROOT}/index.xml" -d "${BUILDLOG_ROOT}"
```

Comment:
`-l` is local time but the server is probably UTC -- this appears to be unnecessary